scripts/utils/options.test.ts

```diff
@@ -35,17 +34,6 @@ const allOptions = createOptions({
   },
 });
 
-// TS "tests"
-// deepscan-disable-next-line
-function test(mv: MaybeOptionValues<typeof allOptions>, v: OptionValues<typeof allOptions>) {
-  console.log(mv.first, mv.second, mv.third, mv.fourth, mv.fifth, mv.sixth);
-  // @ts-expect-error as it's not allowed
-  console.log(mv.seventh);
-  console.log(v.first, v.second, v.third, v.fourth, v.fifth, v.sixth);
-  // @ts-expect-error as it's not allowed
-  console.log(v.seventh);
-}
-
```

Comment:
I don't really think it makes a lot of sense to keep this "test" but if it were really important to prevent these particular cases from being changed this could be wrapped in a trivial test to silence compiler warnings, e.g.
```ts
describe('TS "tests"', () => {
  it('MaybeOptionValues and OptionValues type test', () => {
    // deepscan-disable-next-line
    function test(mv: MaybeOptionValues<typeof allOptions>, v: OptionValues<typeof allOptions>) {
      console.log(mv.first, mv.second, mv.third, mv.fourth, mv.fifth, mv.sixth);
      // @ts-expect-error as it's not allowed
      console.log(mv.seventh);
      console.log(v.first, v.second, v.third, v.fourth, v.fifth, v.sixth);
      // @ts-expect-error as it's not allowed
      console.log(v.seventh);
    }
    expect(test).toBeDefined();
  });
});
```

Reply:
What about using https://github.com/mmkal/expect-type?
We also use that in https://github.com/storybookjs/storybook/blob/e7acb0ed4e5e1bed8274bd53070e753680bba692/code/renderers/react/src/public-types.test.tsx

Reply:
I don't think this test matters that much indeed, so I'm just gonna merge it thanks!